.pre-commit-config.yaml

```
@@ -0,0 +1,23 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.5.0
+    hooks:
+    -   id: trailing-whitespace
+        args: [--markdown-linebreak-ext=md]
+    -   id: end-of-file-fixer
+    -   id: check-toml
+    -   id: check-yaml
+    -   id: check-merge-conflict
+    -   id: check-case-conflict
+    -   id: no-commit-to-branch
+
+-   repo: https://github.com/commitizen-tools/commitizen
+    rev: v3.15.0
```

I think we can use the latest version haha

```suggestion
    rev: v4.6.0
```